test/tools/isoltest.cpp

```diff
@@ -146,6 +152,16 @@ class TestTool
 
 bool TestTool::m_exitRequested = false;
 
+void TestTool::printOptions(ostream& _stream, string const& _linePrefix, bool) const
```

Comment:
Why include `_formatted` if you're not using it? It does not look like you're overriding this function.
```suggestion
void TestTool::printOptions(ostream& _stream, string const& _linePrefix) const
```

Reply:
ops, forgot to delete that ...